src/main/kotlin/com/openlattice/indexing/BackgroundIndexingService.kt

```diff
@@ -333,10 +333,10 @@ class BackgroundIndexingService(
         }
     }
 
-    private fun tryLockEntitySet(esId: UUID): Long? {
+    private fun tryLockEntitySet(esId: UUID): QueueState {
```

Comment:
We are back to Long in develop, so I guess this can be reverted for now

Reply:
Yeah, this is not needed for now